processors/minsev/example_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package minsev // import "go.opentelemetry.io/contrib/processors/minsev"
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	api "go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/sdk/log"
+)
+
+const key = "OTEL_LOG_LEVEL"
+
+var getSeverity = sync.OnceValue(func() api.Severity {
+	conv := map[string]api.Severity{
+		"":      api.SeverityInfo, // Default to SeverityInfo for unset.
+		"debug": api.SeverityDebug,
+		"info":  api.SeverityInfo,
+		"warn":  api.SeverityWarn,
+		"error": api.SeverityError,
+	}
+	// log.SeverityUnknown for unknown values.
```

Comment:
There's no such thing as `log.SeverityUnknown`, so I suppose you mean `log.SeverityUndefined`?